tools/gulp/packaging/rollup-helpers.ts

```diff
@@ -67,5 +72,19 @@ export function createRollupBundle(config: BundleConfig): Promise<any> {
     sourceMap: true
   };
 
+  // When creating a UMD, we want to exclude tslib from the `external` and `globals` bundle options
+  // so that it is inlined into the bundle.
+  if (config.format === 'umd') {
```

Comment:
That was just an example. At some point it will end up similar as with the `convertIndexShorthandImports` in `tsc-wrapped`. 

Anyway, your decision. I don't like it to be honest and this makes the helper into something more magic.